flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/PlanReference.java

```diff
@@ -160,6 +167,43 @@ public String toString() {
         }
     }
 
+    /** Plan reference to a string containing the serialized persisted plan in Smile. */
+    @Experimental
```

Comment:
Make this and the class above PublicEvolving

Reply:
Rename `ContentPlanReference` to `JsonContentPlanReference` and this class to `BytesContentPlanReference`